src/style.css

```diff
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.block1 {
+  background-color: red;
+  height: 100px;
+}
+
+.block2 {
+  background-color: green;
+  height: 100px;
+}
+
+.block3 {
+  background-color: blue;
+  height: 100px;
+}
```

Comment:
- all them have similar property, which can be moved to one general class and this class can be added to all the elements that need this style
- use one more class like `block` that will have similar styles